Xamarin.Forms.Platform.iOS/Renderers/LabelRenderer.cs

```diff
@@ -294,24 +294,21 @@ void UpdateLineBreakMode()
 #endif
 		}
 
-		bool isTextFormatted;
+		FormattedString formatted;
```

Comment:
For our naming standard, this should be `_formatted`.

Reply:
@hartez sure, i will do it
As for naming.. i did it regarding to existing field **_isTextFormatted_** :)